### PR TITLE
Villager related mappings

### PR DESCRIPTION
**Pull request by Platymemo**

### CODE REVIEW

File: mappings/net/minecraft/village/VillagerProfession.mapping [hunk 1]
```diff
@@ -1,5 +1,51 @@
 CLASS net/minecraft/unmapped/C_yhqxfbuc net/minecraft/village/VillagerProfession
 	FIELD f_acjlkvad secondaryJobSites Lcom/google/common/collect/ImmutableSet;
-	FIELD f_ayvuterw gatherableItems Lcom/google/common/collect/ImmutableSet;
+	FIELD f_ayvuterw requestedItems Lcom/google/common/collect/ImmutableSet;
+	FIELD f_kmphlego acquirableJobSite Ljava/util/function/Predicate;
+	FIELD f_obosegzt heldJobSite Ljava/util/function/Predicate;
```
Comment (NoComment1105) [inline mappings/net/minecraft/village/VillagerProfession.mapping]: Is this the job site currently held? If so I think `currentJobSite` would be better?

**Review by NoComment1105 — COMMENTED** [review]
Looking great, I am OneComment today

File: mappings/net/minecraft/village/VillagerProfession.mapping (hunk 1 above)
Comment (Platymemo) [reply mappings/net/minecraft/village/VillagerProfession.mapping]: professions have no notion of whether or not the villager remembers a specific job site, that's up to their brain. This is just a predicate that matches whether the POI they're looking at is a valid job site for them. I think I know some better mappings actually, `primaryJobSites` and `potentialJobSites`

File: mappings/net/minecraft/entity/ai/brain/task/LookTargetUtil.mapping [hunk 2]
```diff
@@ -64,6 +64,10 @@ CLASS net/minecraft/unmapped/C_glrklsny net/minecraft/entity/ai/brain/task/LookT
 		ARG 1 target
 		ARG 2 speed
 		ARG 3 completionRange
+	METHOD m_twcvmjrq throwStack (Lnet/minecraft/unmapped/C_usxaxydn;Lnet/minecraft/unmapped/C_sddaxwyk;Lnet/minecraft/unmapped/C_vgpupfxx;Lnet/minecraft/unmapped/C_vgpupfxx;F)V
+		ARG 0 thrower
+		ARG 2 tossVec
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/entity/ai/brain/task/LookTargetUtil.mapping]: Why use both toss and throw?

Why not `throwVec` and `throwStrength`?

File: mappings/net/minecraft/entity/ai/brain/task/LookTargetUtil.mapping [hunk 3]
```diff
@@ -76,3 +80,6 @@ CLASS net/minecraft/unmapped/C_glrklsny net/minecraft/entity/ai/brain/task/LookT
 			COMMENT the attack target candidate
 		ARG 2 extraDistance
 			COMMENT the max distance this new target can be farther compared to the existing target
+	METHOD m_xafgkpcc throwStack (Lnet/minecraft/unmapped/C_usxaxydn;Lnet/minecraft/unmapped/C_sddaxwyk;Lnet/minecraft/unmapped/C_vgpupfxx;)V
+		ARG 0 thrower
+		ARG 2 tossVec
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/entity/ai/brain/task/LookTargetUtil.mapping]: Same throw/toss question

**Review by supersaiyansubtlety — COMMENTED** [review]
Just questions about throw vs toss.

File: mappings/net/minecraft/village/VillagerProfession.mapping [hunk 4]
```diff
@@ -1,5 +1,51 @@
...
+		ARG 2 primaryJobSites
+		ARG 3 potentialJobSites
+		ARG 4 requestedItems
+		ARG 5 secondaryJobSites
+		ARG 6 workSound
+	METHOD m_afajbkbx register (Ljava/lang/String;Lnet/minecraft/unmapped/C_xhhleach;Lnet/minecraft/unmapped/C_avavozay;)Lnet/minecraft/unmapped/C_yhqxfbuc;
+		ARG 0 id
+		ARG 1 key
+		ARG 2 workSound
+	METHOD m_arfdbrqv heldJobSite ()Ljava/util/function/Predicate;
```
Comment (IotaBread) [inline mappings/net/minecraft/village/VillagerProfession.mapping]: Is this a record?

**Review by IotaBread — APPROVED** [review]
Looks good, aside from the `toss` and `throw` usage mentioned in a comment

File: mappings/net/minecraft/entity/ai/brain/task/LookTargetUtil.mapping (hunk 2 above)
Comment (OroArmor) [reply mappings/net/minecraft/entity/ai/brain/task/LookTargetUtil.mapping]: I would say throwDirection

**Review by OroArmor — APPROVED** [review]

File: mappings/net/minecraft/village/VillagerProfession.mapping (hunk 4 above)
Comment (Platymemo) [reply mappings/net/minecraft/village/VillagerProfession.mapping]: yes, so I'm guessing the method doesn't need mapping? Enigma had greyed names that didn't match the fields, and I'm not sure what that meant so I mapped them

File: mappings/net/minecraft/entity/ai/brain/task/LookTargetUtil.mapping (hunk 2 above)
Comment (Platymemo) [reply mappings/net/minecraft/entity/ai/brain/task/LookTargetUtil.mapping]: good question, `throw...` would be better

**Review by supersaiyansubtlety — APPROVED** [review]

**Review by NoComment1105 — APPROVED** [review]

**Pull request merged**